src/src/com/microsoft/aad/adal/AuthenticationActivity.java

```diff
@@ -776,7 +771,7 @@ protected TokenTaskResult doInBackground(String... urlItems) {
             try {
                 result.taskResult = oauthRequest.getToken(urlItems[0]);
                 Logger.v(TAG, "TokenTask processed the result. " + mRequest.getLogInfo());
-            } catch (Exception exc) {
+            } catch (IOException exc) {
```

Comment:
Nope, let these type of exceptions fall through, because they don't depend on user and if they happen should be fixed by dev